<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Component_management_window</class>
 <widget class="QMainWindow" name="Component_management_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>组件库管理</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,4,2">
    <item>
     <widget class="QWidget" name="widget_1_list" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget_2_base" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="widget_4_scenes" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>12</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_1">
            <property name="text">
             <string>组件预览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4_1" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_4_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>组件操作</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_add">
               <property name="text">
                <string>  新增组件</string>
               </property>
               <property name="icon">
                <iconset resource="component_about.qrc">
                 <normaloff>:/resource/img/add.png</normaloff>:/resource/img/add.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>20</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_import">
               <property name="text">
                <string>  导入组件</string>
               </property>
               <property name="icon">
                <iconset resource="component_about.qrc">
                 <normaloff>:/resource/img/import.png</normaloff>:/resource/img/import.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_change">
               <property name="text">
                <string>  修改组件</string>
               </property>
               <property name="icon">
                <iconset resource="component_about.qrc">
                 <normaloff>:/resource/img/edit.png</normaloff>:/resource/img/edit.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_delete">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>  删除组件</string>
               </property>
               <property name="icon">
                <iconset resource="component_about.qrc">
                 <normaloff>:/resource/img/delete.png</normaloff>:/resource/img/delete.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>22</width>
                 <height>22</height>
                </size>
               </property>
               <property name="autoRepeatDelay">
                <number>300</number>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="component_about.qrc"/>
 </resources>
 <connections/>
</ui>
